tests/components/rfxtrx/test_config_flow.py

```diff
@@ -331,6 +331,9 @@ async def test_no_protocols(hass):
     )
     entry.add_to_hass(hass)
 
+    await hass.config_entries.async_setup(entry.entry_id)
```

Comment:
Here's an example from the config flow tests:
https://github.com/home-assistant/core/blob/452b072bd25c1296c504c24e12bd5a08869f7b39/tests/components/rfxtrx/test_config_flow.py#L58

Reply:
Great. Have added that.

I actually copy/pasted this test to get going, so there are plenty of other places in this test file that don't have this snippet (not added by the other PR, just there already). Do they all need this snippet adding?

I'd love an explanation as to what it's needed for too, I took a look at the dev docs but it didn't really click.

Reply:
Yes, we should always set up the config entry before calling `hass.config_entries.options.async_init`. Otherwise the config flow module may not be loaded if the test is run in isolation. The reason it mostly works now is that other tests in this test module load the config flow module before starting the main flow.

Reply:
Please don't squash or amend commits after review has started. Squashing commits makes it harder for readers to track changes. Thanks!